LandSandBoat/server
# [JAs] Rename BERSERK_EFFECT mod, sort mods, and add notes

**_I affirm:_**

## What does this pull request do?

Rename BERSERK_EFFECT mod to BERSERK_POTENCY in modifier.h, status.lua, and mods_by_id.txt.
Reorganize Warrior mod positions in modifier.h and status.lua lists.
Update job_utils/warrior.lua with renamed mod.
Update item_mods.sql with renamed mod.
Add note to effects/berserk.lua regarding DEFP latent effect from Warrior's Calligae NQ/+1/+2.
Add note to effects/berserk.lua regarding CRIT_HIT_RATE & DOUBLE_ATTACK latent effects from Conqueror (all forms).

This PR is primarily intended as housekeeping and to clarify the mod used by the Conqueror GAxes.
Conqueror i119 III is the only version which does not refer to it as Augments "Berserk".

**References**

https://www.bg-wiki.com/ffxi/Berserk
https://www.bg-wiki.com/ffxi/War._Calligae_%2B2
https://www.bg-wiki.com/ffxi/Conqueror_(Level_119)
https://www.bg-wiki.com/ffxi/Conqueror_(Level_119_III)



## Steps to test these changes



1. Upload changes to your server.
2. Note base ATK and DEF before using Berserk.
3. Use `!setmod 948 8 <me>` to set BERSERK_POTENCY's value to 8 (i119 III Conqueror level).
4. Use Berserk.
5. ATK should be 33% higher and DEF should be 33% lower.
